SETUP_SUBMITTER_CMF.md

```diff
@@ -0,0 +1,120 @@
+- python -m pip install deadline-cloud-for-unreal-engine
+- clone or download deadline-cloud-for-unreal-engine
+
+## Build the Plugin
+
+Adjust the first two paths below based on where your installation of Unreal lives, and where you installed deadline-cloud-for-unreal-engine.
+
+From the Unreal Install Batchfiles Folder (Note the ‘package’ parameter can be any new directory, however you’ll want it to be called “UnrealDeadlineCloudService” later):
+
+- cd C:\Program Files\Epic Games\UE_5.4\Engine\Build\BatchFiles
+- runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src\unreal_plugin\UnrealDeadlineCloudService.uplugin" -package="C:\UnrealDeadlineCloudService"
+- Copy the “package” folder above to your Unreal installation’s Plugins folder (C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService for example)
+
+## (Submitter only) Submitter Installer
+
+Additional python libraries are installed by the submitter installer currently.
+
+- Download submitter installer from Deadline Cloud AWS Console’s Downloads Tab
+- Run, install for all users. Default install location is fine.
+- Enable the Unreal Engine Plugin
+- Make sure the Unreal Engine plugin install path matches where your plugin was copied to (In particular make sure your Unreal version matches)
+
+## (Worker only) pywin32
+
+Unreal’s version of python will need pywin32. Pip install using copy of Unreal’s 3rd Party python installation:
+
+"C:\Program Files\Epic Games\UE_5.4\Engine\Binaries\ThirdParty\Python3\Win64\python" -m pip install pywin32
+
+## Make an AMI (Optional)
+
+If you’ve been setting this instance up as an EC2, it can optionally act as a CMF worker at this point, so you may want to create an AMI at this point.
+
+## Submit a Test Render (Optional)
+
+Assuming you’ve created a CMF and have connected at least one worker that’s had Unreal installed as above, you should be able to submit a test render at this point.
+
+To verify your CMF worker is connected:
+
+On your CMF Worker:
+
+- Open Task Manager
+- Open the Services tab
+- Find “DeadlineWorker”
+  - If you don’t see it listed you’ve likely missed steps (install-deadline-worker in particular) from the CMF host setup steps
+- If the status of the service isn’t currently “Running”, right click it and select start
+- Logs when launching the worker agent to help diagnose installation issues which can cause problems starting the service can be found in C:\ProgramData\Amazon\Deadline\Logs\worker-agent.log\* and C:\ProgramData\Amazon\Deadline\Logs\queue-<queueid>\session-<sessionid>.log
+
+This example will use the Meerkat Demo from the Unreal Marketplace.
+
+- Start the Epic Games Launcher
+- Install the Meerkat Demo from the Unreal Marketplace
+- Create a Project from the Meerkat Demo
+- Open the Project
+- From the Edit Menu, select Plugins, search for and enable UnrealDeadlineCloudService
+- (You may need to restart)
```

Comment:
I believe you *do* need to restart if you haven't enabled the plugin previously.